doc/testing.html

```diff
@@ -594,17 +595,37 @@ <h4 id="macos">MacOS</h4>
 shortcut should be disabled using the steps described above, and then
 deselect "Turn keyboard access on or off" option which is responsible
 for <code>CTRL + F1</code> combination.</p>
-<h4 id="linux">Linux</h4>
+<h5 id="linux">Linux</h5>
 <p>Open the Activities overview and start typing Settings; Choose
 Settings, click Devices, then click Keyboard; set or override desired
 shortcut.</p>
-<h4 id="windows">Windows</h4>
+<h5 id="windows">Windows</h5>
 <p>Type <code>gpedit</code> in the Search and then click Edit group
 policy; navigate to User Configuration -&gt; Administrative Templates
 -&gt; Windows Components -&gt; File Explorer; in the right-side pane
 look for "Turn off Windows key hotkeys" and double click on it; enable
 or disable hotkeys.</p>
 <p>Note: restart is required to make the settings take effect.</p>
+<h4 id="robot-api">Robot API</h4>
+<p>Most part of automated Client UI tests use Robot API to control the
+UI. Usually the default operating system settings need to be adjusted
+for correct work of Robot functionality. The detailed steps how to
+access and update that settings for different platforms are provided
+below.</p>
+<h5 id="macos-1">MacOS</h5>
+<p>Robot functionality is not permitted to control your Mac by default
+starting from MacOS 10.15. To enable it choose Apple menu; System
+Settings, click Privacy &amp; Security; select Accessibility and ensure
+the following apps are allowed to control your computer:
+<code>Java</code> and <code>Terminal</code>. If the tests are run from
+an IDE, the IDE should be granted this permission too.</p>
```

Comment:
You can mention the tests which can be used to verify that the UI, sound, etc are configured properly.(it was done somewhere before one test per option)

Reply:
It is mentioned on an internal wiki page.

More detailed instructions are available at [Automated client GUI testing system set up requirements](https://wiki.openjdk.org/display/ClientLibs/Automated+client+GUI+testing+system+set+up+requirements) wiki page of the Client Libraries.

Reply:
I have added the link to the wiki